validators.js

```diff
@@ -10,6 +10,24 @@ const {
   }
 } = require('./errors');
 
+function validateString(value, name) {
+  if (typeof value !== 'string') {
+    throw new ERR_INVALID_ARG_TYPE(name, 'String', value);
+  }
+}
+
+function validateUnion(value, name, union) {
+  if (!union.includes(value)) {
```

Comment:
Do you need to use an Array primordial for `includes` here for robustness in face of funky prototypes?

Reply:
```suggestion
  if (!ArrayPrototypeIncludes(union, value)) {
```